spec/Provider/AvailableShippingMethodsListProviderSpec.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
```

Comment:
Missing `declare(strict_types=1);` ad return types in methods.

Reply:
Running `bin/ecs check src --fix` should fix all of those.